mathlibtools/lib.py

```diff
@@ -639,6 +639,33 @@ def import_graph(self) -> 'ImportGraph':
         self._import_graph = G
         return G
 
+    def reduce_imports(self, file: str) -> Iterable[List[Tuple[str,str]]]:
```

Comment:
Can you add a docstring indicating what this type contains? I assume the `Tuple[str,str]` are edges (in which direction?), but I have no idea what the intermediate list structure is.

Reply:
```suggestion
    def reduce_imports(self, file: str) -> Iterable[List[Tuple[str, str]]]:
```

Reply:
Done